lib/travis/build/script/matlab.rb

```diff
@@ -30,10 +30,14 @@ def setup
           end
 
           sh.fold 'matlab_install' do
+            sh.echo 'Installing MATLAB system dependencies', ansi: :yellow
```

Comment:
If this is what folks will see for the entire fold, does it make sense to just have 1 message that says something like "Installing MATLAB"?

Reply:
We could do that 🤔

Reply:
What I'm thinking is that if we don't do something like that the fold will say "Installing MATLAB system dependencies" but under that fold it actually installs the system deps and MATLAB.

Reply:
That reasoning makes sense to me. I've updated the PR with that feedback 🤔